proposals/hasCrossSiteAncestor.md

```diff
@@ -0,0 +1,116 @@
+# Proposal: Update Extensions to interact with Partitioned Cookies containing Cross-Site Ancestor Chain Bit values
+
+**Summary**
+
+Allow extensions to utilize a hasCrossSiteAncestor boolean value when interacting with partitioned cookies that include a cross-site ancestor chain bit in their partiton key.
+
+#### Background information: Description of Cross-site Ancestor Chain Bit in partitioned cookies
+
+A cross-site ancestor chain bit is a value that is set by the browser when a partitioned cookie is created. The value is used as part of the key in the partition key to determine access to a partitioned cookie. If the bit indicates true, it means the cookie has been set in a third party context.
+
+A third party context occurs when any parent frame, in the frame tree, is cross-site with the frame or subresource the cookie is being set on. If one site of the frame tree is third party, then all children frames of that frame are also third party and have a cross site ancestor. 
+
+Note: In the table below, sites A1, A2 and A3 are all first party to each other.
+| Site frame tree |Site cookie is set on| hasCrossSiteAncestor value of cookie|
+|---|---|--|
+| A1         |A1| false         | 
+| A1->A2      |A2 | false          | 
+| A1->A2->A3   |A3   | false          | 
+| A1->B       |B| true          |
+| A1->B->A2    |A1   | false          |
+| A1->B->A2 |B    | true          |
+| A1->B->A2 |A2   | true          |
+
+**Document Metadata**
+
+**Author:** aselya 
```

Comment:
Optional, I think it's nice to make these author credits include links to our profiles :)